include/st_pipeline_api.h

```diff
@@ -218,7 +236,7 @@ struct st_frame {
 
   /** priv pointer for lib, do not touch this */
   void* priv;
-  /** priv data for user, get from query_ext_frame callback */
+  /** priv data for user */
   void* opaque;
 };
```

Comment:
We should add iova[ST_MAX_PLANES] for future DMA/HW usage, user can check iova value and if it's not zero then it can be passed to DMA.